### ISSUE TITLE
[Windows] Issues with Heroic with user folder having spaces

### ISSUE THREAD

**Issue by Speedy-VonGofast** [Heroic-Games-Launcher/HeroicGamesLauncher#838]
### Try to get help on our Wiki first: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki/Troubleshooting

**Describe the bug**
Tries to log in indefinitely with a json below:
`{"redirectUrl":"https://epicgames.com/account/personal","authorizationCode":null,"sid":" *hidden* "}`

**Add Logs**

```
INFO:
  Heroic Version: 2.0.0 Roronoa Zoro
  Legendary Version:  0.20.22 Anticitizen One (hotfix #3)
  OS: Microsoft Windows 11 Professionnel KERNEL: 10.0.22000 ARCH: x64
  CPU: Intel® Core™ i7-4790K @4
  RAM: Total: 17.1 GB Available: 9.93 GB
  GRAPHICS: GPU0: NVIDIA GeForce RTX 3060 VRAM: 12288MB DRIVER: 497.09


INFO: User Not Found, removing it from Store
WARNING: Protocol already registered.
[10492:1230/141945.307:INFO:CONSOLE(2)] "MockView: destroy is not supported yet", source: https://static-assets-prod.unrealengine.com/account-portal/static/static/js/2.87a8c929.chunk.js (2)
[10492:1230/141945.536:INFO:CONSOLE(1)] "Refused to apply inline style because it violates the following Content Security Policy directive: "style-src 'self' static-assets-prod.epicgames.com static-assets-prod.unrealengine.com". Either the 'unsafe-inline' keyword, a hash ('sha256-4Su6mBWzEIFnH4pAGMOuaeBrstwJN4Z3pq/s1Kn4/KQ='), or a nonce ('nonce-...') is required to enable inline execution. Note that hashes do not apply to event handlers, style attributes and javascript: navigations unless the 'unsafe-hashes' keyword is present.
", source: https://www.epicgames.com/id/api/redirect (1)
INFO: Logging with Legendary...
ERROR: Error: spawn "C:\Users\Speedy Von Gofast\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary.exe" ENOENT
INFO: Logging with Legendary...
ERROR: Error: spawn "C:\Users\Speedy Von Gofast\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary.exe" ENOENT
[10492:1230/141945.562:INFO:CONSOLE(1)] "Uncaught (in promise) SyntaxError: Unexpected token % in JSON at position 0", source: file:///C:/Users/Speedy%20Von%20Gofast/AppData/Local/Programs/heroic/resources/app.asar/build/index.html#/ (1)
[10492:1230/141945.562:INFO:CONSOLE(1)] "Uncaught (in promise) SyntaxError: Unexpected token % in JSON at position 0", source: file:///C:/Users/Speedy%20Von%20Gofast/AppData/Local/Programs/heroic/resources/app.asar/build/index.html#/ (1)
INFO: checking for heroic updates
[10492:1230/141950.641:INFO:CONSOLE(1)] "Error: Error invoking remote method 'login': reply was never sent", source: file:///C:/Users/Speedy%20Von%20Gofast/AppData/Local/Programs/heroic/resources/app.asar/build/static/js/main.df35c064.chunk.js (1)
ERROR: Frontend: Error: Error invoking remote method 'login': reply was never sent
[10492:1230/141950.641:INFO:CONSOLE(1)] "Error: Error invoking remote method 'login': reply was never sent", source: file:///C:/Users/Speedy%20Von%20Gofast/AppData/Local/Programs/heroic/resources/app.asar/build/static/js/main.df35c064.chunk.js (1)
ERROR: Frontend: Error: Error invoking remote method 'login': reply was never sent
```

**To Reproduce**

1. Just tries to log in on Epic

**Expected behavior**
Logged in and see my list of games

**Screenshots**
https://i.imgur.com/0GollLn.png

**Desktop (please complete the following information):**

- OS: Windows 11
- Heroic Version: 2.0.0

**Additional context**
I can't find a way to log out.

**Comment by flavioislima** [comment]
Hmm, it seems the bug with User Folder with spaces still persists. I thought it was fixed since it's been some time since I didn't hear about it.

can you try the portable version, I think it might work since the legendary binary is inside it.
the portable version is the EXE without the Setup on the name.

**Comment by Speedy-VonGofast** [comment]
I tried the portable version as suggested and it worked.

**Comment by flavioislima** [comment]
Good, unfortunately I could not find a solution for that on Windows yet. So the workaround is to use the portable for now.

**Comment by flavioislima** [comment]
should be fixed now on v2.2.6
